### DISCUSSION TITLE
We can add parcel-plugin-obfuscate, and then how to obfuscate the code?

### DISCUSSION THREAD

**Discussion by nonopolarity** [parcel-bundler/parcel#5937] [Q&A]
I did `yarn build` for simple `index.html` file and all the JS code can be viewed in `dist/index.html`

so then I followed instruction on https://www.npmjs.com/package/parcel-plugin-obfuscate
and did

```
yarn add -D parcel-plugin-obfuscate
```
and now how do I build using the plug in?  (so that the code in `dist/index.html` will become obfuscated.)

I check their doc and the doc on https://parceljs.org/plugins.html

and can't find an answer.

**Comment by DeMoorJasper** [comment]
It probably only works on js files not html files. Parcel doesn't have any need for config so the moment you install a plugin it should work. Also this plugin only supports parcel 1
